Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/TargetsTests/RazorTargetTests.cs

```diff
@@ -100,6 +100,44 @@ public void Razor_PreserveRazorCompilationErrorLog()
             AssertExpectedErrorLog(result, @"C:\UserDefined.json");
         }
 
+        [TestMethod]
+        public void Razor_Prepare_CreatesTempFolderAndPreservesMainFolder()
```

Comment:
ideally I would test also that the subfolder structure is kept, however there's a lot of magic currently in the test orchestration and didn't find a straightforward way to do it, any ideas would be nice

Reply:
I guess we could add a target that would add something into the "0" folder, to give it some structure and we could assert that the structure remained intact after the move.

Reply:
thanks!